code/game/objects/items/stacks/medical_packs.dm

```diff
@@ -268,12 +268,11 @@
 
 //Medical Herbs//
 /obj/item/stack/medical/bruise_pack/comfrey
-	name = "\improper Comfrey leaf"
-	singular_name = "Comfrey leaf"
-	desc = "A soft leaf that is rubbed on bruises."
-	icon = 'icons/obj/hydroponics/harvest.dmi'
-	icon_state = "tea_aspera_leaves"
-	color = "#378C61"
+	name = "\improper Comfrey poultice"
+	singular_name = "Comfrey poultice"
+	desc = "A medical poultice for treating brute injuries, made from crushed comfrey leaves. The effectiveness of the poultice depends on the potency of the comfrey it was made from."
+	icon = 'icons/obj/items.dmi'
```

Comment:
1. Find a place that would fit it, not items.dmi (probably keep it on harvest.dmi??).

2. The old sprites are still being used by some other item?

Reply:
1. Made medical.dmi and put all the medical items items.dmi in there.
2. Old sprites were reused from the harvest items of comfrey and aloe (not the same item, new sprites do not belong in harvest.dmi at all).

Reply:
If you really want to move all the medical sprites here then i would suggest you to spawn all of them to test, i think you you forgot to change the icon path in some items.